test/distributed/test_dynamo_distributed.py

```diff
@@ -386,6 +391,39 @@ def fn():
             pass
         self.assertEqual(res, 1)
 
+    @patch.object(config, "optimize_ddp", False)
+    def test_ignored_parameters(self):
+        """
+        Verifies ddp graph-split logic ignores parameters marked to ignore on DDP module.
+        Hooks up graph-split optimizer manually so it can peek at internal state.
+        """
+        m, inputs, correct_outputs = get_custom_model(self.device)
+        parameters_to_ignore = ["seq.2.weight", "seq.4.linear.bias"]
+        DDP._set_params_and_buffers_to_ignore_for_model(m, parameters_to_ignore)
+        ddp_m = DDP(m, device_ids=self.device_ids, bucket_cap_mb=25)
+        parameter_ids_to_ignore = [
+            id(ddp_m.module.get_parameter(p))
```

Comment:
so.. this seems better than the hacky fqn/mangled name thing.  But, is it totally reliable?  I wondered if there could be edge cases, or if dynamo would possibly make copies, etc.

cc @voznesenskym @zdevito